Makefile

```diff
@@ -166,7 +166,7 @@ endif
 	MDD_BENCHMARK_PRECISION=1 \
 	$(ASAN_COMMAND) $(PYTHON) -m pytest demos --nbmake $(PARALLELIZE)
 
-wheel:
+wheel: standalone-plugin
```

Comment:
This is interesting because there aren't any other explicit dependencies either, it just assumes everything has been built already. Maybe the mlir target should build standalone automatically instead?

Reply:
I've just added it because there is a failure when `make wheel` is invoked before `make all`. But if it is assumed `make wheel` must only be used when everything has been built already, there is no point on adding it anywhere, I would say.
Our GitHub workflows are built under that assumption. I will just delete it.

Reply:
Removed.